tests/test_client_v2_integration.py

```diff
@@ -119,10 +129,16 @@ def test_url_input_source_must_not_raise_errors(
     """
     url = os.getenv("MINDEE_V2_SE_TESTS_BLANK_PDF_URL")
 
-    input_doc = UrlInputSource(url)
-    options = InferenceParameters(findoc_model_id)
+    input_source = UrlInputSource(url)
+    params = InferenceParameters(
+        model_id=findoc_model_id,
+        rag=False,
+        raw_text=False,
+        polygon=False,
+        confidence=False,
+    )
```

Comment:
```suggestion
    params = InferenceParameters(
        model_id=findoc_model_id,
        rag=False,
        raw_text=False,
        polygon=False,
        confidence=False,
        webhook_ids=[],
        alias='py_integration_url_source'
    )
```

Just noticed we never tested aliases for URLs.